File: 20220810/모범답안/20220810_04.js/josephusProblem.py
def solution(N, K):
    arr = [i for i in range(1, N+1)]
    number = 0
    for _ in range(N - 1):
        number += (K - 1)
        number %= len(arr)
        arr.remove(arr[number])

    return arr[0]

File: 20220810/모범답안/20220810_04.js/josephusProblem.js
function solution(N, K){
    var  arr = new Array(N);
    var number = 0;
  
    for(var i = 0; i < N; i++){
      arr[i] = i + 1;
    }
  
    for(var i = 0; i < N - 1; i++){
      number += (K - 1);
      number %= arr.length;
  
      arr.splice(number, 1);
    }
  
    return arr[0];
  }